### DISCUSSION TITLE
How to make sidebar fixed size ?

### DISCUSSION THREAD

**Discussion by edgarriba** [pradyunsg/furo#268] [Questions] [answered]
HI, I was wondering if it's possible to fix the size of the sidebar.

I would like to stick to the sidebar always to left side to avoid the blue space (see below)

![image](https://user-images.githubusercontent.com/5157099/135118243-6c42e083-43de-4e4e-bd5a-e663b79b1176.png)

**Comment by pradyunsg** [comment, marked as answer]
This isn't possible. The theme is designed to center the content on wide screens, and this is baked into the scaffolding CSS.
